packages/core/src/screens/LoginScreen.js

```diff
@@ -76,6 +78,9 @@ const LoginScreen = ({route}) => {
   }, [modeDebug, releaseInstanceConfig?.showUrlInput]);
 
   const defaultUrl = useMemo(() => {
+    if (urlStorage !== undefined) {
+      return urlStorage;
+    }
```

Comment:
It's better to check for undefined and null values by doing != null instead of !== undefined

```
    if (urlStorage != null) {
      return urlStorage;
    }
```